### DISCUSSION TITLE
Add landmark recognition

### DISCUSSION THREAD

**Discussion by ralfi** [immich-app/immich#5182] [Q&A]
Hi there, first a big "thanks" for immich. 

Playing around a few weeks as alternative for nextcloud photos and/or memories i find immich very useful and fast. Congrats! My question: It is possible to integrate or extend the machine learning with landmark/sights recognation LLMs like https://www.kaggle.com/models/google/landmarks?tfhub-redirect=true, using with nextcloud?

Regards,
ralfi

**Comment by jrasm91** [comment]
If the have geo data, we can also just use that cc: @zackpollard :laughing:

**Comment by pstumpe** [comment]
+1
also coming from Nextcloud I find this very useful - especially for old scanned photos
